docs/designers-developers/developers/block-api/block-patterns.md

```diff
@@ -15,16 +15,19 @@ The `register_block_pattern` function receives the name of the pattern as the fi
 The properties of the pattern include:
  - `title` (required): A human-readable title for the pattern.
  - `content` (required): Raw HTML content for the pattern.
+ - `description` (required): A description of the pattern. Used to describe the pattern in the inserter.
```

Comment:
I'm personally hesitant about making this required. I think it shouldn't be as a lot of existing patterns don't have it and we should fallback to the title. Sometimes the title is descriptive enough.

Reply:
Is there a way to not have it be required but be really really encouraged?

Reply:
mmm I don't know, we could console log a message if we don't find it maybe but we never did that before and it can be very verbose. I can't think of a way though.

Reply:
Is compatibility for existing patterns the only concern? _Because the function was just renamed,_ developers had to make changes then, why is adding a description a bigger concern?

Reply:
I've thought about this more and I still think it shouldn't be mandatory. Some patterns are descriptive enough and the API should be as straightforward to use as possible.

Enforcing on a potential Patterns directory/repository is a separate question though and I'd see this being different there.

Reply:
But I thought that was the point, the patterns are not descriptive if you can't see them.
And currently, neither the title or the content is required, even though the documentation says they are.
Without a title, the only thing announced by screen readers is "button".

Adding a description is more straight forward than adding categories (especially if you throw in registering the categories).